plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -290,6 +293,16 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
                                     const transaction_metadata_ptr& trx,
                                     bool api_trx, bool return_failure_trace,
                                     next_function<transaction_trace_ptr> next );
+      push_result handle_push_result( const transaction_metadata_ptr& trx,
+                                      next_function<transaction_trace_ptr> next,
+                                      const fc::time_point& start,
+                                      const chain::controller& chain,
+                                      const transaction_trace_ptr& trace,
+                                      bool return_failure_trace = true,
+                                      bool disable_subjective_enforcement = false,
+                                      account_name first_auth = {},
+                                      int64_t sub_bill = 0,
+                                      uint32_t prev_billed_cpu_time_us = 0);
```

Comment:
I would remove all these defaults and be explicit.

Reply:
I was worried the parameter list was too long. Default parameters removed, so the code intention is more clearer.